impeller/entity/save_layer_utils_unittests.cc

```diff
@@ -280,77 +280,18 @@ TEST(SaveLayerUtilsTest,
   EXPECT_EQ(coverage.value(), Rect::MakeLTRB(0, 0, 50, 50));
 }
 
-TEST(SaveLayerUtilsTest,
```

Comment:
Why are all of these other test cases deleted? I think it's important to verify that we are rounding up in reasonable cases but not in "unreasonable" cases (and though our policy on unreasonable might change over time, not having any test for it means it could change unintentionally)

Reply:
I added a few more test cases that cover one or both of the dimensions being too different for the round up.